alerts/utilization.libsonnet

```diff
@@ -14,8 +14,8 @@
               severity: 'warning',
             },
             annotations: {
-              message: 'Storage cluster is nearing full. An expansion is required',
-              description: 'The utilization of storage cluster has crossed 85%.',
+              message: 'Storage cluster is nearing full. Expansion is required.',
```

Comment:
May be we can write "Storage cluster expansion is required"

Reply:
I feel `Storage cluster` twice in same message doesn't look good. I will still go with existing message.